CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,94 @@
+
+These descriptions are intended to share understanding of prohibited conduct.  The descriptions are not presented or intended to be exhaustive lists.  
+
+### C. THE COMPLAINT PROCESS
+
+Any person electing to utilize this complaint resolution procedure will be treated courteously.   The complaint will be handled as confidentially as possible.  Apereo will take appropriate action to the extent possible to address any violations of this policy.   Groundless, untruthful, or malicious complaints will result in comparable sanctions.
+
+The following complaint procedure will be followed in order to address a complaint regarding harassment.
+
+1. Apereo will designate two Board members to receive complaints.  At Apereo Foundation events, two members of the Board or the event committee will be designated to receive complaints.  Designated individuals are called “Duty Officers.”   
+2. A person who feels harassed may report any concerns to the Duty Officers.
+3. Upon receiving the complaint, the Duty Officer will thorough review the reported activities and provide a detailed review of the situation to the Apereo Executive Director.  If the Apereo Executive Director is unavailable, the Board Chair or a named proxy will act in place.
+4. If the complaint is determined to have merit, the Apereo Executive Director, Chair, or named proxy will decide and execute appropriate sanctions, possibly including but not limited to:
+   1. Verbal warning.
+   2. Removal from the event without refund or removal from a working group.
+   3. Prohibition from attending future events or participating in other working groups.
+
+Nothing in this policy shall prevent the complainant or the respondent from pursuing formal legal remedies or resolution through appropriate agencies or the courts.
+
+## 2. Event Policy
+
+### Apereo Event and Open Apereo Conference Code of Conduct
+
+The Apereo Foundation events are working conferences and meetings intended to present perspectives, develop an understanding of other academic enterprises, organize community-wide projects, promote collaboration, and develop partnerships in the Apereo community. Attendees are expected to behave according to internationally expected professional standards, in accordance with local relevant laws and customs, and in accordance with their employer's expectations for appropriate workplace behavior.
+
+While at Apereo events or related social networking opportunities, speakers, presenters, and attendees should not engage in speech or actions that may be considered harassment or unwelcoming.
+
+The Apereo Foundation reserves the right take appropriate action, to warn an offender, to deny entrance to an offender, or to expel an offender without refund.
+
+We value your attendance at this event and want your experience to be positive and welcoming.
+
+Please report any concerns to the immediate attention:
+
+1. Duty Officer:  Name, contact info
+2. Duty Officer:  Name, contact
```

Comment:
Duty officers need names and contact info

Reply:
While [this oversight was escalated to the Board in June 2017|https://groups.google.com/a/apereo.org/d/msg/uportal-dev/hJas1FdYW1w/xdiXrct0AQAJ], it is still not yet resolved.

Arguably uPortal should withdraw the `CODE_OF_CONDUCT.md` until this is resolved. We're not doing anyone any favors claiming to have a code of conduct that isn't actually operationalized and viable.